test.js

```diff
@@ -1,56 +1,75 @@
 'use strict'
```

Comment:
I would add a dumb test that doesn't pass any arguments that expect the `ConfigurationError('Missing node(s) option')`